Documentation/Signoffs/Microcontroller.md

```diff
@@ -0,0 +1,51 @@
+3. External storage support is required of this subsystem. The most preferred form of external storage is a Universal Serial Bus (USB) Flash Drive as it is common in households and easily available.
+4. A major goal of the project is to allow for synchronous operations of at least 3 air quality sensors.  Sensors may be either analog or digital, so there must be support for both.  The board must have enough Input/Output (I/O) pins to operate any combination of 3 sensors whether they be digital or analog.   Different sensors interface with a potentially different number of pins.  Preliminary research shows that a sensor may take up to 5 pins to operate.  Other sensors may have more, but it is more common for 5 and under.  This means the board must have at least 15 digital input pins and 15 analog input pins.
+5. The cpu clock speed does not need to be very fast.  The code will be mostly Input/Output heavy with less computation.  This means that there will be a lot of waiting for responses and does not require a high-speed processor.  A cpu clock speed of at least 1 MHz is preferred for fast bootup.  Cpu's over 1 MHz are very common and just provides a reliable speed for what computation is present.
+6. There must be sufficient memory for the code.  It is unknown how much memory the code will occupy.  Comparing similar operating code with libraries and compilation show slightly over or under 150 KB (Kilobytes) in Flash/Disk/Storage.  To account for the worst case, the board will need over 150 KB.
+7. The power requirements of the board need to be reasonably under or near 240 mW.  Here, we take a voltage supply of 12 V and current of 20 mA to get 240 mW.  12 V with 20 mA is from the power subsystem’s output to provide an adequate operation time or lifespan and not damaged components.  Going over 240 mW could risk draining the battery too quickly.
+8. Some form of battery monitoring capabilities need to be included or applicable.  
+9. The board itself cannot be too big.  “Too big” here is defined as having a volume greater than 50 cubic inches.  This size was set to prevent the device from proving too large.  50 cubic inches would be about the size of a small novella/book.  .  
+10. The manufacturer needs to be a reputable and safe choice.  A stable manufacturere that is assumed to remain in business for years to come and continue producing the chosen board is a necessecity.  This project aims for replicability, so a board that remains relevant and available is a key part.
+11. The board will need to have coding language support for C and or Python.  Support for both is preferred for options and coverage but not required.  It is expected that the users will have some degree of coding experience and may potentially personalize or improve upon the code. These languages are chosen due to popularity and performance.  C, in particular, is predicted to be very desirable for this project.
```

Comment:
If the language is a constraint, which language is the required language?

Reply:
We want the code to be looked at and potentially have very small changes made to it by less-experienced users.  C and Python have a plethora of examples and guides.  We will specify that either language is okay, but at least one must be supported.